src/QuickCheckVEngine/RVFI_DII/DII.hs

```diff
@@ -115,3 +122,15 @@ diiEnd :: DII_Packet
 diiEnd = DII_Packet { dii_cmd  = dii_cmd_end
                     , dii_time = 1
                     , dii_insn = 0 }
+
+-- | Construct a version negotiation 'DII_Packet'
```

Comment:
Can we add in the comment something like "A version negotiation packet uses a 'dii_cmd_end' command with embedded metadata to guarantee that implementations unable to take part in negotiations carry on unperturbed"?

Reply:
Sure sounds good.